project/.sbtserver

```diff
@@ -0,0 +1,3 @@
+#Server Startup at 2015-06-16T11:17+0000
```

Comment:
Do we need to have this file in Github? (and `.sbtserver.lock`, which I can't comment on because it's empty)